packages/web-app-admin-settings/src/components/Users/SideBar/DetailsPanel.vue

```diff
@@ -81,6 +103,21 @@ export default defineComponent({
       return this.$gettext(
         this.roles.find((role) => role.id === assignedRole.appRoleId)?.displayName || ''
       )
+    },
+    groupsDisplayValue() {
+      return this.user.memberOf
+        .map((group) => group.displayName)
+        .sort()
+        .join(', ')
+    },
+    quotaDisplayValue() {
+      if (!('total' in (this.user.drive?.quota || {}))) {
```

Comment:
The edit panel shows a message that the quota is only available after the user has logged in at least once. Can we do the same here please to keep it consistent?

Reply:
yes